openfl-workspace/xgb_higgs/src/setup_data.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import os
+import shutil
+from logging import getLogger
+from urllib.request import urlretrieve
+from hashlib import sha384
+from os import path, makedirs
+from tqdm import tqdm
+import modin.pandas as pd
+import gzip
+from sklearn.model_selection import train_test_split
+import numpy as np
+
+logger = getLogger(__name__)
+
+"""HIGGS Dataset."""
+
+URL = "https://archive.ics.uci.edu/ml/machine-learning-databases/00280/HIGGS.csv.gz"
+FILENAME = "HIGGS.csv.gz"
+CSV_FILENAME = "HIGGS.csv"
+CSV_SHA384 = 'b8b82e11a78b81601381420878ad42ba557291f394a88dc5293e4077c8363c87429639b120e299a2a9939c1f943b6a63'
+DEFAULT_PATH = path.join(path.expanduser('~'), '.openfl', 'data')
```

Comment:
I suggest creating within same directory, instead of a hidden folder. This folder may not be guaranteed to mount in an enclave (one less thing to worry about if a user ever wants to do that)